Source/Fuse.Scripting/Tests/ScriptClass/ScriptPromiseTests.uno

```diff
@@ -85,11 +85,11 @@ namespace Fuse.Scripting.Test
 		public void FailTest()
 		{
 			var child = new UX.BasicTest();
+			var p = new Promise<double>();
+			p.Reject(new Exception("fail"));
```

Comment:
Maybe you should do this inside the `using` block, to better simulate when it might happen?

Reply:
I mean just the call to `Reject`, not the whole setup.

Reply:
Yep 👍